alr.gpr

```diff
@@ -12,8 +12,9 @@ project Alr is
    Src_Dirs := ("src/alr");
 
    case Alire_Common.Host_Os is
-      when "Windows_NT" => Src_Dirs := Src_Dirs & ("src/alr/os_windows");
-      when others       => Src_Dirs := Src_Dirs & ("src/alr/os_linux");
+      when "windows" => Src_Dirs := Src_Dirs & ("src/alr/os_windows");
+      when "osx"     => Src_Dirs := Src_Dirs & ("src/alr/os_macos");
```

Comment:
Why `osx` and not `macos` here?

Reply:
I do see what you mean: `alr_env.gpr` sets `GNATCOLL_OS` to `osx`, & I just followed that for `ALIRE_OS`.